README.md

```diff
@@ -1,15 +1,22 @@
 # Ploigos GitHub Workflows
 
-In order to use this workflow for your application, you must do the following -
+This repository houses the Ploigos workflows defined within a Github Actions [reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows). For ease of use, there are templates that can be used within the examples directory. Please replace all values that begin with `REPLACE_`.
 
-Create a psr.yaml file in your application repository
+## Setup Instructions
```

Comment:
Let's call this something more descriptive like "Using this Workflow in Your Application"

Reply:
this is done